Sorting/Bubble Sort/bubblesort.py

```diff
@@ -0,0 +1,12 @@
+def bubble_sort(list1):  
+    for i in range(0,len(list1)-1):  
+        for j in range(len(list1)-1):  
+            if(list1[j]>list1[j+1]):  
+                temp = list1[j]  
+                list1[j] = list1[j+1]  
+                list1[j+1] = temp  
+    return list1  
+  
+list1 = [5, 3, 8, 6, 7, 2]  
```

Comment:
Hey pal, your code looks good .😄
Points to change 
- Remove the code from line 10 (Just need the algorithm).
- Give a description about bubble sort as well as about its complexity .